rstudio/rhel9-python-3.9/Dockerfile

```
@@ -5,6 +5,7 @@ FROM ${BASE_IMAGE}
 ARG SECRET_DIR=/opt/app-root/src/.sec
 ARG SERVERURL_DEFAULT=""
 ARG BASEURL_DEFAULT=""
+ARG INSTALL_ADD_PKGS=""
```

In order to avoid the installation by default, we should define this as:
```suggestion
ARG INSTALL_ADD_PKGS=false
```
And we will need to refine the logic in the line 76 (Check there for the refine)